video.go

```diff
@@ -58,6 +59,9 @@ const defaultPadding = 72
 const defaultPlaybackSpeed = 1.0
 const defaultWidth = 1200
 
+// DefaultStartingFrame is the index of the file to start the image sequence loop from
+const DefaultStartingFrame = 1
```

Comment:
Can we make this unexported similar to the rest?

Reply:
I think I can. Let me push a commit to address this.